guides/security-oidc-identity-propagation.adoc

```diff
@@ -0,0 +1,400 @@
+= Identity Propagation with OpenID Connect
+:summary: Learn how to propagate identities within a deployment and across deployments when securing WildFly apps with OpenID Connect.
```

Comment:
perhaps remove 'WildFly' here.
On my Lenovo/Fedora/Chrome env the box for this one on the guides page is 'taller' than the others so the layout is not precise. I *think* removing this word would fix it. Oddly, when I change my local checkout and rebuild, nothing changes, but that must be some error on my part!

Reply:
I removed 'WildFly' from the description for this guide. Depending on the description the boxes are always slightly different in height.